proxy/README.md

```diff
@@ -0,0 +1,12 @@
+# Keycloak Security Proxy
+
+The Keycloak Security Proxy docker image.
+
+## Usage
+
+This image is intended to be used with an existing `/tmp/conf/proxy.json` on the host machine. 
+See [the proxy documentation](https://keycloak.gitbooks.io/documentation/server_installation/topics/proxy.html) 
+for more details.
```

Comment:
@jpkrohling here's my quick attempt only to check if the docker image run.

1. Put proxy.json inside /tmp/conf. This is the config file: https://gist.github.com/abstractj/d4a10027670d698d31230f031d0a8672
2. Run `docker run --rm --name keycloak-proxy -v /tmp/conf:/opt/jboss/conf -p 8180:8180 jpkroehling/keycloak-proxy`

I get this error:

```
Home directory: /opt/jboss/keycloak-proxy
Jul 19, 2017 9:10:27 AM org.keycloak.proxy.ProxyServerBuilder initConnections
WARN: Generating temporary SSL cert
Exception in thread "main" java.lang.reflect.InvocationTargetException
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at org.keycloak.Launcher.main(Launcher.java:81)
Caused by: java.lang.RuntimeException: org.keycloak.common.util.PemException: java.security.spec.InvalidKeySpecException: encoded key spec not recognised
        at org.keycloak.adapters.KeycloakDeploymentBuilder.internalBuild(KeycloakDeploymentBuilder.java:69)
        at org.keycloak.adapters.KeycloakDeploymentBuilder.build(KeycloakDeploymentBuilder.java:164)
        at org.keycloak.proxy.ProxyServerBuilder$ApplicationBuilder.<init>(ProxyServerBuilder.java:164)
        at org.keycloak.proxy.ProxyServerBuilder.application(ProxyServerBuilder.java:129)
        at org.keycloak.proxy.ProxyServerBuilder.build(ProxyServerBuilder.java:397)
        at org.keycloak.proxy.ProxyServerBuilder.build(ProxyServerBuilder.java:378)
        at org.keycloak.proxy.Main.main(Main.java:41)
        ... 5 more
Caused by: org.keycloak.common.util.PemException: java.security.spec.InvalidKeySpecException: encoded key spec not recognised
        at org.keycloak.common.util.PemUtils.decodePublicKey(PemUtils.java:84)
        at org.keycloak.adapters.KeycloakDeploymentBuilder.internalBuild(KeycloakDeploymentBuilder.java:65)
        ... 11 more
Caused by: java.security.spec.InvalidKeySpecException: encoded key spec not recognised
        at org.bouncycastle.jcajce.provider.asymmetric.util.BaseKeyFactorySpi.engineGeneratePublic(Unknown Source)
        at org.bouncycastle.jcajce.provider.asymmetric.rsa.KeyFactorySpi.engineGeneratePublic(Unknown Source)
        at java.security.KeyFactory.generatePublic(KeyFactory.java:328)
        at org.keycloak.common.util.DerUtils.decodePublicKey(DerUtils.java:66)
        at org.keycloak.common.util.PemUtils.decodePublicKey(PemUtils.java:82)
        ... 12 more
```
Am I supposed to do anything different?